ui/packages/consul-ui/app/components/topology-metrics/status/index.hbs

```diff
@@ -1,12 +1,16 @@
-{{#if @noMetricsReason}}
+{{#if (or @noMetricsReason @error)}}
   <span class="topology-metrics-error">
-    Unable to load metrics
-    <span>
-      <Tooltip>{{@noMetricsReason}}</Tooltip>
-    </span>
+    {{#if (eq @noMetricsReason 'ingress-gateway')}}
+      {{t "components.consul.topology-metrics.status.ingress-gateway"}}
+    {{else if @error}}
+      {{t "components.consul.topology-metrics.status.error"}}
+    {{else if (eq @noMetricsReason 'remote-dc')}}
+      {{t "components.consul.topology-metrics.status.error"}}
+      <span
+        {{tooltip (t "components.consul.topology-metrics.status.remote-dc")}}
```

Comment:
I figured out what was wrong with this tooltip. It needed to be tucked inside the `span`.